### PR TITLE
Add section on subscribing to the handbook + badge

### PR DESCRIPTION
**Pull request by michaelpj**

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Introduction
 
 This document forms the Cardano Engineering Handbook, which aims to provide documentation and policy that applies to all projects in the Cardano Open Source Consortium (COSC).
-To avoid confusion, we recommend that if a project is following this Handbook, then it should link to the Handbook in its `README`.
+To avoid confusion, projects should [explicitly subscribe](policy/project/index.html) to the Handbook.
```
Comment (abailly-iohk) [inline introduction.md:4]: shouldn't `should` be `SHOULD`?

**Review by abailly-iohk — APPROVED** [review]
Minor typo

**Review by coot — APPROVED** [review]

File: introduction.md (hunk 1 above)
Comment (michaelpj) [reply introduction.md:4]: Yep!

Comment (michaelpj) [reply introduction.md:4]: Or wait, no. This is indeed the non-SHOULD should, because this is just some prose, not a policy :D

Comment (abailly-iohk) [reply introduction.md:4]: 🤯

**Pull request merged**